### ISSUE TITLE
Note block occlusion

### ISSUE THREAD

**Issue by Cupboard-Nose** [daipenger/Sound-Physics#19]
Any way to have note blocks "ignore" intervening blocks, the same way the jukeboxes are currently implemented?

Also, thank you for picking up this project. Sound has too long been the underdog to graphics in the gaming world.

**Comment by Cupboard-Nose** [comment]
Having experimented a bit more I now realise that jukeboxes don't behave quite how i thought. They actually seem to take the acoustic conditions from when they start playing and keep them, regardless of where you go afterwards (presumably because each song is a single "sound"). They aren't exempt from absorption like i thought.

**Comment by daipenger** [comment]
Yep, Jukeboxes behave this way. And another reason to completely rethink this mod and give it a total rewrite...
What I could do, is filter out all noteblock/jukebox sounds and let them play with the OpenAL defaults/Minecraft default.
